tensorflow/compiler/xla/service/gpu/elemental_ir_emitter.cc

```diff
@@ -178,7 +178,7 @@ StatusOr<llvm::Value*> GpuElementalIrEmitter::EmitFloatBinaryOp(
   switch (op->opcode()) {
     case HloOpcode::kRemainder: {
 
-      return EmitLibdeviceMathCall("fmod", {lhs_value, rhs_value},
+      return EmitLibdeviceMathCall("__ocml_fmod", {lhs_value, rhs_value},
```

Comment:
will `__ocml_fmod` work for the CUDA path too? or will this change break the CUDA path?
same question applies for the similar change in PR 395.

Reply:
It wont work for CUDA path. Our XLA path is currently hardcoded in many places for rocm.  I have two branches - starting with branch device-functions- that are for making both CUDA and rocm paths co-exist. As soon as PR 396 is merged will test the first one.